_stable/server/hello-world.md

```diff
@@ -126,6 +126,24 @@ async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
 
 To see all the snippets put together, check out the [full example][example]!
 
+This example uses the `http1` module to create a server that speaks HTTP/1.
+In case you want to use HTTP/2, you can use the `http2` module with a few changes to the
+http1 server. The http2 builder requires an executor to run. Executor should implement the `hyper::rt::Executor` trait.
+
+```rust
+impl<F> hyper::rt::Executor<F> for TokioExecutor
```

Comment:
Instead of this code example inline, you can link to the `../init/runtime.md` guide.